manifest.ttl.in

```diff
@@ -5,4 +5,10 @@
 <http://gareus.org/oss/lv2/@LV2NAME@>
 	a lv2:Plugin ;
 	lv2:binary <@LV2NAME@@LIB_EXT@>  ;
-	rdfs:seeAlso <@LV2NAME@.ttl> .
+	rdfs:seeAlso <@LV2NAME@.ttl>, <modgui.ttl> .
```

Comment:
This must only ever be used for the MOD version, not the desktop variant.
Other x42-plugins use `sed` to inject MOD specifics when building the turtle files.